kubernetes/cmsweb/scripts/deploy-cm.sh

```diff
@@ -6,11 +6,29 @@ if [ $# -ne 3 ]; then
     exit 1
 fi
 
+cluster_name=`kubectl config get-clusters | grep -v NAME`
+
 ns=$1
 srv=$2
 conf=$3
 
+tmpDir=/tmp/$USER/sops
+if [ -d $tmpDir ]; then
+   rm -rf $tmpDir
+fi
+mkdir -p $tmpDir
+cd $tmpDir
+
+if [ -z "`command -v sops`" ]; then
+  # download soap in tmp area
+  wget -O sops https://github.com/mozilla/sops/releases/download/v3.7.2/sops-v3.7.2.linux.amd64
```

Comment:
Since we download new script in user's home/bin area I suggest to put appropriate message here, e.g.
`echo "Download and install sops scripts under $HOME/bin"`